discovery/db_discovery.go

```diff
@@ -193,10 +236,15 @@ func ethOrchToDBOrch(orch *lpTypes.Transcoder) *common.DBOrch {
 	if orch == nil {
 		return nil
 	}
-	return &common.DBOrch{
-		ServiceURI:   orch.ServiceURI,
-		EthereumAddr: orch.Address.String(),
+
+	dbO := &common.DBOrch{
```

Comment:
Can we just return the struct literal here?

Reply:
fixed in d9c1a2d9